src/group_theory/sylow.lean

```diff
@@ -600,6 +600,21 @@ begin
   exact P.1.card_subgroup_dvd_card,
 end
 
+/-- A subgroup with cardinality `p ^ n` is a Sylow group
```

Comment:
```suggestion
/-- A subgroup with cardinality `p ^ n` is a Sylow subgroup
```

Reply:
This wording was consistent with the documentation of [sylow.card_eq_multiplicity](https://leanprover-community.github.io/mathlib_docs/group_theory/sylow.html#sylow.card_eq_multiplicity), it might make sense to change "Sylow group" to "Sylow subgroup" there as well (and of [sylow.direct_product_of_normal](https://leanprover-community.github.io/mathlib_docs/group_theory/sylow.html#sylow.direct_product_of_normal)).